index.js

```diff
@@ -31,6 +33,44 @@ async function run() {
     }
     containerDef.image = imageURI;
 
+    if (environmentVariables) {
+
+      // If environment array is missing, create it
+      if (!Array.isArray(containerDef.environment)) {
+        containerDef.environment = [];
+      }
+
+      // Get pairs by splitting on newlines
+      environmentVariables.split('\n').forEach(function (line) {
+        // Trim whitespace
+        const trimmedLine = line.trim();
+        // Skip if empty
+        if (trimmedLine.length === 0) { return; }
+        // Split pairs on =
+        const pair = line.trim().split('=');
+        // If that pair didn't have exactly 2 parts (name and value)
+        if (pair.length != 2) {
```

Comment:
Sorry I keep realizing small issues 😅 What would happen if a user has this env var:
```
  NAME=val1=val2
``` 
I wonder if the logic should be:
```js
const separatorIdx =  trimmedLine.indexOf("=");
if (separatorIdx === -1) {
   throw new Error(...);
}
const variable = {
  name: trimmedLine.substring(0, separatorIdx),
  value: trimmedLine.substring(separatorIdx + 1),
}
```